daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/GroupDef.scala

```diff
@@ -141,7 +141,7 @@ sealed abstract class GlobalGroupDef(
     if (dais.nonEmpty)
       SDW(
         WarnID.InvalidAnnotationPoint,
-        "Annotations placed directly on a group definition will be ignored by DFDL. Any annotation expected to be processed by DFDL should instead be placed on the group reference, sequence or choice."
+        "Annotations placed directly on a group definition will be ignored by DFDL. Any annotation expected to be processed by DFDL should instead be placed on the group reference, sequence or choice.",
```

Comment:
If this comma is truly required, then our scalafmt rules are wrong.

This comma should be tolerated, ... maybe... but not required. 

This is not a list, it's the arguments to a function call. They are fundamentally different concepts that only occasionally overlap.